app/cdap/components/Administration/AdminTabSwitch/index.tsx

```diff
@@ -94,7 +94,7 @@ export default class AdminTabSwitch extends React.PureComponent<IAdminTabSwitchP
   public renderUptimeVersion() {
     return (
       <span className="uptime-version-container">
-        <span>
+        <span data-testid="system-uptime-container" data-uptime={this.props.uptime}>
```

Comment:
what is this **data-uptime** attribute added for? newly added one it seems

Reply:
Yes, it's added to verify that the uptime value (as an integer) is passed correctly to the UI irrespective of the localisation.